packages/create-svelte/cli/modifications/add_typescript.js

```diff
@@ -13,19 +13,37 @@ import {
  *
  * @param {string} cwd
  * @param {boolean} yes
+ * @param {boolean} is_skeleton_template
  */
-export default async function add_typescript(cwd, yes) {
+export default async function add_typescript(cwd, yes, is_skeleton_template) {
```

Comment:
there was a question about whether we could reverse this to start with the typescript template and make it `remove_typescript`. we have `svelte-preprocess` which should already offer the ability to strip out the typescript stuff and would make it so that we don't have to update this script at all when we change the template. using `svelte-preprocess` might be a big change to include in this PR, but it might be a step towards that to at least reverse the logic even if still done manually